core/bin/run

```diff
@@ -2,6 +2,14 @@
 
 #### Housekeeping functions and variables ####################################
 
+if [[ $(date +%u) -eq 3 && $(date +%H) -eq 1 ]]; then
```

Comment:
Let's add a comment up top here explaining what this does and why for future reference!